.changes/unreleased/tinymce-TINY-11077-2024-07-26.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Caret would unexpectedly go to a not editable table row above when user pressed enter.
```

Comment:
Suggested improvement: from `Caret would unexpectedly go to a not editable table row above when user pressed enter.` to `Caret would unexpectedly shift to the non-editable table row above when pressing Enter.`